packages/grid/_modules_/grid/components/toolbar/DensityMenuSelector.tsx

```diff
@@ -2,15 +2,15 @@ import * as React from 'react';
 import Button from '@material-ui/core/Button';
 import MenuItem from '@material-ui/core/MenuItem';
 import ListItemIcon from '@material-ui/core/ListItemIcon';
+import { densityValueSelector } from '../../hooks/features/density/densitySelector';
 import { useIcons } from '../../hooks/utils/useIcons';
+import { Density, DensityTypes } from '../../models/density';
 import { ApiContext } from '../api-context';
-import { DensityTypes, Density } from '../../models/gridOptions';
 import { useGridSelector } from '../../hooks/features/core/useGridSelector';
 import { DensityOption } from '../../models/api/densityApi';
-import { densityValueSelector } from '../../hooks/features/density';
 import { GridMenu } from '../menu/GridMenu';
 
-export function DensitySelector() {
+export function DensityMenuSelector() {
```

Comment:
Do we need to rename this? Is there a convention for it? If we rename this then that means we should update the docs because now we have different terminology for it, on one place we call it Density selector but in the code, we call it DensityMenuSelector.

Reply:
I can see the following

```jsx
    <GridToolbar>
      {!options.disableColumnSelector && <ColumnsToolbarButton />}
      {!options.disableColumnFilter && <FilterToolbarButton />}
      {!options.disableDensitySelector && <DensityMenuSelector />}
    </GridToolbar>
```

Reply:
How about?

```jsx
    <GridToolbar>
      {!options.disableColumnSelector && <GridToolbarColumns />}
      {!options.disableColumnFilter && <GridToolbarFilter />}
      {!options.disableDensitySelector && <GridToolbarDensity />}
    </GridToolbar>
```

Reply:
I will rollback!
I didn't like the name selector as we use the prefix selector to select part of the state 🤔